OpenRA.Mods.Common/UtilityCommands/UpgradeMapCommand.cs

```diff
@@ -38,16 +48,16 @@ static void ProcessYaml(ModData modData, Map map, MiniYaml yaml, int engineDate,
 				var files = FieldLoader.GetValue<string[]>("value", yaml.Value);
 				foreach (var filename in files)
 				{
-					var fileNodes = MiniYaml.FromStream(map.Open(filename), filename);
+					var fileNodes = MiniYaml.FromStream(Open(filename, package, modData.DefaultFileSystem), filename);
 					processYaml(modData, engineDate, ref fileNodes, null, 0);
 
 					// HACK: Obtain the writable save path using knowledge of the underlying filesystem workings
 					var packagePath = filename;
-					var package = map.Package;
+					var p = package;
```

Comment:
Any reason why you don't use the package parameter directly here instead of the p variable?

Reply:
This is inside a loop, and each file needs access to the original `package` value.